src/commands/alerts.ts

```diff
@@ -8,9 +8,9 @@ import { OutputStream } from "../lib/output-stream";
 
 const pageSize = 100; // The max.
 
-async function listAlerts(query: string, offset: number, apiKey: string, options: any): Promise<any[]> {
```

Comment:
`apiKey` was actually not used in the method (it is part of the Options argument).